blog-cse/2023-04-28-content.md

```diff
@@ -0,0 +1,70 @@
+---
+title: April 28, 2023 - Content Release
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+#### Rules
+Updates several Azure based rules to account for modifications made to normalization mappers.
+
+* [New] MATCH-S00873 AWS EKS Cluster Configuration Updated; AWS EKS clusters contain various configuration options, including for which IP addresses can access the cluster API. Ensure that this change is authorized and expected.
+* [New] MATCH-S00872 AWS EKS Failed Curl Authentication Attempt; Failed instances of curl usage within a containerized environment should occur rarely, investigate the source IP address used to ensure that it is legitimate.
+* [New] MATCH-S00871 AWS EKS Pod Shared Object Modification or Creation; A Kubernetes pod was either created, updated or patched with a shared process namespace.
+* [New] MATCH-S00870 AWS EKS Secrets Created; Kubernetes secrets may be created for legitimate purposes, ensure that this secrets created is from an IAM account that is expected to manage Kubernetes workloads on EKS.
```

Comment:
```suggestion
* [New] MATCH-S00870 AWS EKS Secrets Created: Kubernetes secrets may be created for legitimate purposes. Ensure that the secret created is from an IAM account that is expected to manage Kubernetes workloads on EKS.
```

Reply:
I found another instance of "this secrets created is". Change both to "the secrete created is".